en-US/firefox/all.lang

```diff
@@ -0,0 +1,104 @@
+## NOTE: The page is at https://www-dev.allizom.org/firefox/all/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/all/
+
+
+# %s refers to Firefox.
```

Comment:
I would probably say "%s is replaced with 'Firefox'", but it's also OK to keep it like this